>>> 'Pkg.add("Orchestra")' log
INFO: Cloning cache of DecisionTree from git://github.com/bensadeghi/DecisionTree.jl.git
INFO: Cloning cache of DimensionalityReduction from git://github.com/JuliaStats/DimensionalityReduction.jl.git
INFO: Cloning cache of Orchestra from git://github.com/svs14/Orchestra.jl.git
INFO: Cloning cache of PyCall from git://github.com/stevengj/PyCall.jl.git
INFO: Installing ArrayViews v0.6.2
INFO: Installing DataArrays v0.2.16
INFO: Installing DataFrames v0.6.7
INFO: Installing Dates v0.4.4
INFO: Installing DecisionTree v0.3.8
INFO: Installing DimensionalityReduction v0.1.1
INFO: Installing Docile v0.5.10
INFO: Installing FactCheck v0.2.8
INFO: Installing GZip v0.2.17
INFO: Installing Iterators v0.1.8
INFO: Installing MLBase v0.5.1
INFO: Installing Orchestra v0.0.5
INFO: Installing PyCall v0.8.1
INFO: Installing Reexport v0.0.2
INFO: Installing SortingAlgorithms v0.0.5
INFO: Installing StatsBase v0.6.15
INFO: Package database updated

>>> 'Pkg.test("Orchestra")' log
Julia Version 0.4.0-dev+5846
Commit 0503f2a (2015-07-08 02:04 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
INFO: Testing Orchestra
Warning: New definition 
    call(Type{PyCall.PyDict}, Base.Associative{#K<:Any, Any}) at /home/vagrant/.julia/v0.4/PyCall/src/conversions.jl:464
is ambiguous with: 
    call(Type{PyCall.PyDict}, Base.Associative{Any, #V<:Any}) at /home/vagrant/.julia/v0.4/PyCall/src/conversions.jl:463.
To fix, define 
    call(Type{PyCall.PyDict}, Base.Associative{Any, Any})
before the new definition.
ERROR: LoadError: LoadError: LoadError: LoadError: TypeError: apply_type: in first parameter of NTuple, expected Int64, got Type{Type{T}}
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:133
 in reload_path at ./loading.jl:157
 in _require at ./loading.jl:69
 in require at ./loading.jl:55
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:133
 in include at ./boot.jl:254
 in include_from_node1 at ./loading.jl:133
 in reload_path at ./loading.jl:157
 in _require at ./loading.jl:69
 in require at ./loading.jl:52
 in include at ./boot.jl:254
 in include_from_node1 at loading.jl:133
 in process_options at ./client.jl:306
 in _start at ./client.jl:406
while loading /home/vagrant/.julia/v0.4/PyCall/src/PyCall.jl, in expression starting on line 381
while loading /home/vagrant/.julia/v0.4/Orchestra/src/system.jl, in expression starting on line 4
while loading /home/vagrant/.julia/v0.4/Orchestra/src/Orchestra.jl, in expression starting on line 5
while loading /home/vagrant/.julia/v0.4/Orchestra/test/runtests.jl, in expression starting on line 4
==============================[ ERROR: Orchestra ]==============================

failed process: Process(`/home/vagrant/julia/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.4/Orchestra/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
INFO: No packages to install, update or remove
ERROR: Orchestra had test errors
 in error at ./error.jl:21
 in test at pkg/entry.jl:746
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in test at pkg.jl:71
 in process_options at ./client.jl:282
 in _start at ./client.jl:406

>>> End of log
